transformer_lens/components.py

```diff
@@ -377,7 +409,7 @@ def __init__(
         else:
             raise ValueError(f"Invalid attention type: {self.attn_type}")
 
-        self.register_buffer("IGNORE", torch.tensor(-1e5))
+        self.register_buffer("IGNORE", torch.tensor(torch.finfo(cfg.dtype).min))
```

Comment:
It may be constroversial but I modified the value of the layer "IGNORE" from `-1e-5` to `torch.finfo(cfg.dtype).min`. This shouldn't change much in practice, I just thought it was cleaner and closer to what Hugging Face did. I can revert that if needed.

Reply:
I'll revert that since it is done in the PR #319.